### DISCUSSION TITLE
Discussion Topics for Sep 20, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#2250] [Show and tell]
Discussion Topics for Sep 20, 2021 Engineering Sync

**Comment by whytheplatypus** [comment]
How we get from here to there, ideas on getting to the next iterations of report stream.
- config syntax (settings, schemas, whatever you want to call them).. goal of extreme clarity (sets us up for external parties using it later)
- potential transform pipeline: e.g. <parser> -> sender validation -> sender transforms -> FHIR models -> non-sender specific transforms -> receiver filters -> receiver transforms -> receiver serializers

Ways to start?
- Possibly start with only hl7 paths through as a simple first step for FHIR.
- Separate concerns (schemas + logic) for sender and receiver configuration.

**Comment by carlosfelix2** [comment]
Development process - quick run down
